app/jobs/job_posting_poll_job.rb

```diff
@@ -2,8 +2,13 @@
 
 class JobPostingPollJob < ApplicationJob
   queue_as :default
+  sidekiq_options retry: 2
 
   def perform(*)
-    JobPostingApi::Poller.new.perform
+    JobPosting.without_auto_index do
+      JobPostingApi::Poller.new.perform
+    end
+
+    JobPosting.reindex!
   end
```

Comment:
Why do you first deactivate and then force indexing?

Reply:
Because Algolia automatically indexes one item when changing it. The poller would then cause 4000 single indexing jobs, which obviusli iz not gud. Hence, I disable auto indexing, perform the job, and then index all at once

Reply:
I expected this, just wanted to make sure me understood what it was doing